docs/src/user/basic_setup.md

```diff
@@ -47,11 +47,11 @@ Install from the GitHub release.
 Note that this way also requires the aforementioned installation.
 
 ```console
-$ wget https://github.com/containers/youki/releases/download/v0.3.0/youki_0_3_0_linux.tar.gz
-$ tar -zxvf youki_0_3_0_linux.tar.gz youki_0_3_0_linux/youki-0.3.0/youki
+$ wget https://github.com/containers/youki/releases/download/v0.3.0/youki_0_3_0_linux-$(uname -m).tar.gz
+$ tar -zxvf youki_0_3_0_linux-$(uname -m).tar.gz
 # Maybe you need root privileges.
-$ mv youki_0_3_0_linux/youki-0.3.0/youki /usr/local/bin/youki
-$ rm -rf youki_0_3_0_linux.tar.gz youki_0_3_0_linux
+$ mv youki_0_3_0_linux-$(uname -m)/youki-0.3.0/youki /usr/local/bin/youki
+$ rm -rf youki_0_3_0_linux-$(uname -m).tar.gz youki_0_3_0_linux-$(uname -m)
 ```
```

Comment:
Could you keep the method before `v0.3.0`  using `<details>`?
Like that:
<details><summary> version < v0.3.0</summary>
<p>

```console
$ wget https://github.com/containers/youki/releases/download/v0.3.0/youki_0_3_0_linux.tar.gz
$ tar -zxvf youki_0_3_0_linux.tar.gz youki_0_3_0_linux/youki-0.3.0/youki
# Maybe you need root privileges.
$ mv youki_0_3_0_linux/youki-0.3.0/youki /usr/local/bin/youki
$ rm -rf youki_0_3_0_linux.tar.gz youki_0_3_0_linux
```

</p>
</details>